test-data/unit/check-isinstance.test

```diff
@@ -1443,6 +1443,28 @@ else:
 [builtins fixtures/isinstancelist.pyi]
 
 
+[case testIssubclasDestructuringUnions]
+from typing import Union, List, Tuple, Dict, Type
+def f(x: Union[Type[int], Type[str], Type[List]]) -> None:
```

Comment:
@pkch 

Something strange happened with typeshed. Could you please fix this?
(And don't forget to open an issue about ``Type[Union[...]]`` vs ``Union[Type[...]]``.)